src/Facades/Qr.php

```diff
@@ -267,8 +309,22 @@ public static function output(?string $data = null, ?array $options = null): Htm
             reset($options['logo']);
             $logo = current($options['logo']);
 
-            if (filled($logo->getPathName())) {
-                $maker = $maker->merge($logo->getPathName(), .4, true);
+
+            if ($logo instanceof \Illuminate\Http\UploadedFile && filled($logo->getPathName())) {
```

Comment:
I am not following this part and why we need it?
if the image loaded from a storage path I dont think we need to upload it again, right?